au4/src/appshell/internal/applicationactioncontroller.cpp

```diff
@@ -73,16 +72,17 @@ void ApplicationActionController::onDragEnterEvent(QDragEnterEvent* event)
 
 void ApplicationActionController::onDragMoveEvent(QDragMoveEvent* event)
 {
-    const QMimeData* mime = event->mimeData();
-    QList<QUrl> urls = mime->urls();
-    if (urls.count() > 0) {
-        const QUrl& url = urls.front();
-        if (projectFilesController()->isUrlSupported(url)
-            || (url.isLocalFile() && audio::synth::isSoundFont(io::path_t(url)))) {
-            event->setDropAction(Qt::LinkAction);
-            event->acceptProposedAction();
-        }
-    }
+    //! TODO AU4
+    // const QMimeData* mime = event->mimeData();
+    // QList<QUrl> urls = mime->urls();
+    // if (urls.count() > 0) {
+    //     const QUrl& url = urls.front();
+    //     if (projectFilesController()->isUrlSupported(url)
+    //         || (url.isLocalFile() && audio::synth::isSoundFont(io::path_t(url)))) {
+    //         event->setDropAction(Qt::LinkAction);
+    //         event->acceptProposedAction();
+    //     }
+    // }
```

Comment:
Let's add task about open project by file drag and drop  (here https://github.com/orgs/audacity/projects/27/views/1)